gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -660,3 +660,9 @@ api:
   properties:
     encryption:
       secret: vvLJ4Q8Khvv9tm2tIPdkGEdmgKUruAL6
+
+portal:
```

Comment:
We do not need to add this to the `gravitee.yml`. There should be documentation written by technical writers to explain how the user can add these settings to the `gravitee.yml` or in a cloud environment.

Reply:
Done